test-app/package.json

```diff
@@ -24,8 +23,7 @@
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
     "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\" --prefixColors auto",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
```

Comment:
Dropped this because we don't want it to be included when doing `pnpm test`.
We run the scenarios individually in the workflow